api/everest.go

```diff
@@ -330,3 +335,37 @@ func (e *EverestServer) k8sToAPIErrorHandler(next echo.HTTPErrorHandler) echo.HT
 		next(err, c)
 	}
 }
+
+func enforcerErrorHandler(next echo.HTTPErrorHandler) echo.HTTPErrorHandler {
+	return func(err error, c echo.Context) {
+		if errors.Is(err, errInsufficientPermissions) {
+			err = &echo.HTTPError{
+				Code:    http.StatusForbidden,
+				Message: errInsufficientPermissions.Error(),
+			}
+		}
+		next(err, c)
+	}
+}
+
+// enforceOrErr is a wrapper around casbin.Enforce that returns an error if the enforcement fails.
+func (e *EverestServer) enforceOrErr(subject, resource, action, object string) error {
```

Comment:
I don't like the name `enforceOrErr` very much. Most functions in go return error and we don't call all of them `xOrErr` we call them just `x` :sweat_smile: